cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
```

Comment:
Coverity found issue: Missing TLS - CWE-319, High Severity

****

**How to fix:** Explicitly set the `Properties.DomainEndpointOptions.EnforceHTTPS` value to `true` to require HTTPS for all connections.

<details>
<summary>Click to expand data flow...</summary>

**From cft/deploy.yml:**
```
    1 Resources:
    2   ElasticsearchDomain:
    3     Type: AWS::Elasticsearch::Domain
1. Sigma main event: The Amazon Elasticsearch Service (Amazon ES) domain does not enforce HTTPS connections, exposing data in transit.

2. remediation: Explicitly set the `Properties.DomainEndpointOptions.EnforceHTTPS` value to `true` to require HTTPS for all connections.

    4     Properties:
    5       DomainName: "test"
    6       ElasticsearchClusterConfig:
```
</details><!-- Coverity 2d4c223daa215e11639eca98f7aaff50 -->